x/accesscontrol/types/keys.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+const (
+	// ModuleName defines the module name
+	ModuleName = "accesscontrol"
+
+	QuerierRoute = ModuleName
+
+	StoreKey = ModuleName
+)
```

Comment:
this is where we would add in the message dependency mapping key builders so that we can store it independently in the keeper